src/goto-programs/resolve_concrete_function_call.cpp

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************\
+
+ Module: Goto Program utilties
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#include <algorithm>
+
+#include "resolve_concrete_function_call.h"
+
+/// See the operator() method comment.
+/// \param symbol_table: The symbol table to resolve the function call in
+resolve_concrete_function_callt::resolve_concrete_function_callt(
+  const symbol_tablet &symbol_table):
+    symbol_table(symbol_table)
+{
+  class_hierarchy(symbol_table);
```

Comment:
I've provided both as options - in one case we have no use for the `class_hierarchy` but in the other 2 we already have one.